src/tide-tables/header.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import glamorous from 'glamorous'
+import Styles from '../assets/styles'
+import moment from 'moment'
+
+export default class extends Component {
+  get formattedDate() {
+    return moment(this.props.date).format('dddd, MMM D')
```

Comment:
thoughts on making the date format a constant? eg: `const DATE_FORMAT` at the top of the component?

Reply:
Yup, agreed. I am going to do that and also a few more for the keys that im using for the request and `localStorage`